### PR TITLE
Doc-535 Reworked Stream.md Topic

### PR DESCRIPTION
**Pull request by diana-macrometa**
@StoyanTc, please read through this carefully. Some of the content, especially in Stream Source and Stream Sink, sounded a bit odd to me. For example, I could not find an XML mapper like they talked about.

**What I did:**
- Reviewed stream.md, now renamed CREATE STREAM. Also renamed Trigger to CREATE TRIGGER.
- Moved content for Stream Source into the Source folder.
- Moved content for Stream Sink into the Sink folder.

### CODE REVIEW

**Comment by diana-macrometa** [comment]
@StoyanTc, will you be able to review this PR this week?

File: docs/cep/source/stream-source.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+The following is the list of source mapping types supported by stream.
+
+|Source mapping type | Description|
+| ------------- |-------------|
+| [CSV](../query-guide/functions/sourcemapper/csv.md) | Converts CSV-like delimiter separated events to stream events.|
+| [JSON](../query-guide/functions/sourcemapper/json.md) | Converts JSON messages to stream events.|
+| [Key-Value](../query-guide/functions/sourcemapper/keyvalue.md) | Converts key-value hash maps to stream events.|
+| [PassThrough](../query-guide/functions/sourcemapper/passThrough.md) | Omits data conversion on stream events.|
+| [Text](../query-guide/functions/sourcemapper/text.md) | Converts plain text messages to stream events.|
+| XML | Converts XML messages to stream events.|
```
Comment (StoyanTc) [inline docs/cep/source/stream-source.md]: @diana-macrometa - We don't support the `XML`.

File: docs/cep/source/stream-source.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+```
+
+The stream worker extends the above use case by adding failure generation and error handling with the use of queries is as follows.
+
+```sql
+-- Define fault stream to handle error occurred at TempStream subscribers
+
+CREATE STREAM TempStream WITH(OnError.action="STREAM") (deviceID long, roomNo int, temp double;
+
+-- Error generation through a custom function `createError()`
```
Comment (StoyanTc) [inline docs/cep/source/stream-source.md]: @diana-macrometa - Let' take this out, we don't have such custom function.

**Review by StoyanTc — COMMENTED** [review]
@diana-macrometa - Here are the missing descriptions for `Source` and `Sink`. They are erroneously explained only for streams but actually are the general ones.

File: docs/cep/source/stream-source.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/cep/source/stream-source.md]: We kind of do through the JavaScript functions, but I agree, let's stick to basic functions here.

Comment (StoyanTc) [reply docs/cep/source/stream-source.md]: No, this is not a JS function. The description is for a custom extension, so called `stream processor`.

Comment (diana-macrometa) [reply docs/cep/source/stream-source.md]: Did I remove the wrong part? I thought I removed the custom function bit you were referring to.

**Review by StoyanTc — APPROVED** [review]
@diana-macrometa- 
It's ok now.
What do you plan for `stream-source` and `stream-sink` since those are the common descriptions?

**Comment by diana-macrometa** [comment]
@StoyanTc, I'm going to compare them to the OTHER common descriptions that I just found buried in a tutorial and try to cobble something coherent into the intro topics for Sink and Source.

**Pull request merged**